zxlive/edit_panel.py

```diff
@@ -79,20 +77,15 @@ def _vert_double_clicked(self, v: VT) -> None:
         if self.graph.type(v) == VertexType.BOUNDARY:
             return
 
-        input, ok = QInputDialog.getText(
+        input_, ok = QInputDialog.getText(
             self, "Input Dialog", "Enter Desired Phase Value:"
         )
         if not ok:
             return
         try:
-            new_phase = Fraction(input)
+            new_phase = Fraction(input_)
         except ValueError:
-            msg = QMessageBox()
-            msg.setIcon(QMessageBox.Critical)
-            msg.setText("Wrong Input Type")
-            info_text = "Please enter a valid input (e.g. 1/2, 2)"
-            msg.setInformativeText(info_text)
-            msg.exec_()
+            show_error_msg("Wrong Input Type", "Please enter a valid input (e.g. 1/2, 2)")
```

Comment:
I noticed that this diaglog disappears immediately (even before I made any changes). Can any of your reproduce this?

After some investigating it seems like the `QMessageBox` breaks if it is put after `QInputDialog.getText(...)`